dice.js

```diff
@@ -0,0 +1,10 @@
+
+const rollDice = function(){
+    const numOne = Math.round(Math.random() * 6) + 1; 
+    const numTwo = Math.round(Math.random() * 6) + 1;
+    const sum = numOne + numTwo;
+
+    console.log('Dice rolled are ${numOne} and ${numTwo}. Sum is ${sum}')
```

Comment:
this prints out Dice "rolled are ${numOne} and ${numTwo}. Sum is ${sum}", if you want to show variables value in a string correctly use ` instead of '